src/androidProj/nativeeditplugin/src/main/java/com/bkmin/android/EditBox.java

```diff
@@ -233,10 +233,10 @@ public void Create(int _tag, JSONObject jsonObj)
                 case "IntegerNumber" : editInputType |= InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_NORMAL; break;
                 case "DecimalNumber" : editInputType |= InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_FLAG_DECIMAL; break;
                 case "Alphanumeric" : editInputType |= InputType.TYPE_CLASS_TEXT  | InputType.TYPE_TEXT_FLAG_CAP_SENTENCES | InputType.TYPE_TEXT_VARIATION_NORMAL; break; // This is default behaviour
-                case "Name" : editInputType |= InputType.TYPE_TEXT_VARIATION_PERSON_NAME; break;
+                case "Name" : editInputType |= InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_PERSON_NAME; break;
                 case "EmailAddress" : editInputType |= InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS; break;
                 case "Password" : editInputType |= InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_PASSWORD; break;
-                case "Pin" : editInputType |= InputType.TYPE_TEXT_VARIATION_PHONETIC; break;
+                case "Pin" : editInputType |= InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_PHONETIC; break;
```

Comment:
I know it was broken originally, but I just spotted that `TYPE_TEXT_VARIATION_PHONETIC` for `Pin` is incorrect. I think we could use `TYPE_CLASS_PHONE` for it, just like iOS:
https://github.com/kmbang/UnityNativeEdit/blob/ed0d364b8d101bc85397d141666376575ed75d79/demo/Assets/NativeEditPlugin/Plugins/iOS/EditBox_iOS.m#L240